### PR TITLE
CIRCLE-5368 - Add Nomad builder for CCIE 2.0

### PR DESCRIPTION
**Pull request by MikeyYeahYeah**

### CODE REVIEW

File: nomad/tests/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+- hosts: localhost
+  become_method: sudo
+  connection: local
+  roles:
+    - { role: nomad }
+#  vars:
+#    this_role_varname: 'some value'
```
Comment (christophermancini) [inline nomad/tests/playbook.yml]: You can just remove this instead of commenting it out.

File: nomad/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- name: add docker package repo
+  become: yes
+  apt_repository:
+    repo: deb [arch=amd64] https://download.docker.com/linux/ubuntu {{ ansible_lsb.codename}} stable
+
+- name: install docker-ce
+  become: yes
+  apt:
+    name: docker-ce
+    update_cache: yes
```
Comment (christophermancini) [inline nomad/tasks/main.yml]: All of the docker install stuff can be removed if you add my docker role to the `meta/main.yml` file like it is in `builder/meta/main.yml`.

File: nomad/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```yaml
+---
+- hosts: localhost
+  become_method: sudo
+  connection: local
+  roles:
+    - { role: this_role }
+  vars:
+    this_role_varname: 'some value'
+```
```
Comment (christophermancini) [inline nomad/README.md]: Update this file! UX ftw!

**Comment by MikeyYeahYeah** [comment]
@cmancini  Changes pushed.  I will squash after you approve of changes

**Pull request merged**